inst/qml/Upgrades.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick		2.12
```

Comment:
The `Upgrades.qml` file should be under `inst/`, not `inst/qml/`... please move it there :)